src/auth/Signup.js

```diff
@@ -0,0 +1,109 @@
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import BaseUrl from '../redux/base_url';
+
+export default function Login() {
+  const navigate = useNavigate();
+  const fldClass = 'block w-full px-4 py-2 mt-2 text-clrPrime bg-white border rounded-md focus:border-clrPrime focus:ring-clrPrime focus:outline-none focus:ring focus:ring-opacity-20';
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const name = e.target[0].value;
+    const email = e.target[1].value;
+    const password = e.target[2].value;
```

Comment:
:red_circle:  [REQUIRED]  I think it is considered a best practice:star:  to validate :heavy_check_mark:  your input :arrow_forward:  on the client side :desktop_computer:  before sending :outbox_tray:  to the server as the client-side validation :white_check_mark:  provides a better user experience :woman_dancing: :man_dancing:  by responding quickly at the browser level. You have done well by setting some HTML5 attributes such as required, and type of the fields but you forgot to set a minimum acceptable character length for fields like password and name. Alternatively, you can use javascript to validate your inputs